fs/spiffs/src/spiffs_core.c

```diff
@@ -1534,8 +1534,10 @@ void spiffs_fobj_event(FAR struct spiffs_s *fs,
   FAR struct spiffs_file_s *next;
   int16_t objid = objid_raw & ~SPIFFS_OBJID_NDXFLAG;
 
+#ifdef CONFIG_DEBUG_FS_INFO
```

Comment:
why need add ifdef?

Reply:
to match the above variable declaration.

Reply:
how about remove the CONFIG_DEBUG_FS_INFO check at line 1526? since many other similar place normally remove the check.

Reply:
done